manifests/init.pp

```diff
@@ -121,7 +121,7 @@
 #
 class simp (
   Hash                            $scenario_map,
-  Enum['simp', 'simp_lite', 'poss', 'none'] $scenario         = 'simp',
+  Simp::Scenario                  $scenario                   = 'simp',
```

Comment:
These need to be listed out so that they are obvious to the users. Hiding them behind a Data Type still makes them too difficult to find as a distinct list.